server/app/views/ApplicantUtils.java

```diff
@@ -1,10 +1,31 @@
 package views;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
 import java.util.Optional;
+import play.i18n.Lang;
 import play.i18n.Messages;
+import play.i18n.MessagesApi;
 import services.MessageKey;
 
 public final class ApplicantUtils {
+
+  private final MessagesApi messagesApi;
+
+  @Inject
+  public ApplicantUtils(MessagesApi messagesApi) {
+    this.messagesApi = checkNotNull(messagesApi);
+  }
+
+  /** Get the applicant's name or the GUEST message with en-US lang. */
+  public String getApplicantNameEnUs(Optional<String> maybeName) {
+    Messages messages = messagesApi.preferred(ImmutableList.of(Lang.forCode("en-US")));
```

Comment:
Will this need to be updated for intl?

Reply:
Only if we ever overhaul the admin UI to support i18n.